### PR TITLE
website: changes for 1.6.0 beta

### PR DESCRIPTION
**Pull request by pearkes**
Still TODO:

- [x] Update homepage for additional features
- [x] Update use-cases for additional features
- [x] Fill in broken links (fixing CI)

### CODE REVIEW

**Comment by pearkes** [comment]
CI is failing as there are still a few links to fill in that we haven't created yet.

File: website/source/redirects.txt [hunk 1]
```diff
@@ -49,6 +49,7 @@
 /docs/guides/bootstrapping.html            /docs/install/bootstrapping.html
 /docs/guides/sentinel.html                 /docs/agent/sentinel.html
 /docs/connect/proxies/sidecar-service.html /docs/connect/registration/sidecar-service.html
+/docs/enterprise/connect-multi-datacenter/index.html /docs/enterprise/index.html
```
Comment (banks) [inline website/source/redirects.txt]: Will this just leave old link going somewhere confusing? Do we need a placeholder page that is not in the menu bar but just explains that replication etc. are now OSS and that advanced federation etc. are enterprise or something?

I can see why it is a little awkward to maintain that given the "not calling it multi-dc anymore" stance and probably not enough links here to care that much but just a thought?

File: website/source/docs/connect/connect-internals.html.md [hunk 2]
```diff
@@ -87,16 +87,44 @@ agent may begin failing and eventually crash. Cache entries do have TTLs
...
+without externally routable IPs at the service level.
+
+## Intention Replication
+
+Intention replication happens automatically but requires the
+[`primary_datacenter`](/docs/agent/options.html#primary_datacenter)
+configuration to be set to specify a datacenter that is authoritative
+for intentions. In production setups with ACLs enabled, the
+[replication token](/docs/agent/options.html#acl_tokens_replication) must also
+be set in secondary datacenter server's configuration.
```
Comment (banks) [inline website/source/docs/connect/connect-internals.html.md]:
```suggestion
be set in the secondary datacenter server's configuration.
```

File: website/source/redirects.txt (hunk 1 above)
Comment (pearkes) [reply website/source/redirects.txt]: I went with the "probably not enough links here to care that much" – I figured we didn't need to special-case it in any particular way. I considered redirecting to the connect internals page but that felt confusing as well.

**Comment by pearkes** [comment]
I've added changes for the marketing pages of the website. You can view them live [here](https://5d1ff5e3f5016d8f625ac9d9--consul-docs-preview.netlify.com/).

**Pull request merged**